{"name": "新增用户", "status": "passed", "attachments": [{"name": "log", "source": "02e24438-6a37-4350-bda3-f7dd6ef9f4ea-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "inData", "value": "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'"}, {"name": "exp", "value": "'{'code':500,'msg':\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'"}], "start": 1605080496390, "stop": 1605080496416, "uuid": "a96d120d-142f-4a36-ba65-1d5873b1a0dd", "historyId": "457e3c2b06bcc702a9859bd245944053", "testCaseId": "dc3f5c27b8f716b793706de44b0c3c3f", "fullName": "test_UserManage.TestUserManage#test_add_user", "labels": [{"name": "suite", "value": "test_UserManage"}, {"name": "subSuite", "value": "TestUserManage"}, {"name": "host", "value": "dadaxPC008"}, {"name": "thread", "value": "1252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_UserManage"}]}